### PR TITLE
feat: gift wrap data into agol-pallet-shelf folder

### PR DESCRIPTION
**Pull request by stdavis**
This should prevent data from being pushed with the FORKLIFT_HASH field to AGOL.

### CODE REVIEW

File: sgid/AGOLPallet.py [hunk 1]
```diff
@@ -55,15 +56,37 @@ def build(self, config):
             parcels_gdb = r'\\{}\c$\MapData\Parcels.gdb'.format(prod_server['machineName'])
             self.add_crate([statewide_parcels_name, parcels_gdb, share_fgdb])
 
+    def process(self):
+        self.log.debug('cleaning up output data folder')
+        output_data_folder = join(dirname(forklift_config.get_config_prop('hashLocation')), 'agol-pallet-shelf')
+
+        try:
+            rmtree(output_data_folder)
+            mkdir(output_data_folder)
```
Comment (steveoh) [inline sgid/AGOLPallet.py]: do these need to be in two different try's?

if trying to delete the folder fails, will the creation of the folder be skipped and cause an issue later on?

Comment (stdavis) [reply sgid/AGOLPallet.py]: My thought was that if the delete fails because of it already existing, then `mkdir` is unneeded.

Comment (steveoh) [reply sgid/AGOLPallet.py]: so deleting a folder that does not exist doesn't raise an error?

Comment (stdavis) [reply sgid/AGOLPallet.py]: I'm a dope.

**Comment by stdavis** [comment]
> does gift wrap or something else create the folder making my other comment not important?

No, gift-wrap actually just removes the hash field and compresses. `copy_with_overwrite` is executed on each FGDB. So we do need to manually create the folder.

**Pull request merged**